### PR TITLE
RFC feedforward max_rate_limit and jitter_factor

### PR DESCRIPTION
**Pull request by haslinghuis**
Needs firmware from https://github.com/betaflight/betaflight/pull/11017

Screenshot from 2021-11-18 02-59-51

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Checked and tested, works perfectly, thank you @haslinghuis !

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by klutvott123 — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

File: src/tabs/pid_tuning.html [hunk 1]
```diff
@@ -605,12 +605,19 @@
...
+                                        <input type="number" name="feedforwardJitterFactor" step="1" min="0" max="20"/>
                                         <label>
-                                            <span i18n="pidTuningFeedforwardTransition"></span>
+                                            <span i18n="pidTuningFeedforwardJitter"></span>
+                                        </label>
+                                        <div class="helpicon cf_tip" i18n_title="pidTuningFeedforwardJitterHelp"></div>
+                                    </span>
+
+                                    <span class="feedforwardOption feedforwardSmoothFactor suboption">
+                                        <input type="number" name="feedforwardSmoothFactor" step="1" min="0" max="75" />
```
Comment (haslinghuis) [inline src/tabs/pid_tuning.html:617]: @ctzsnooze max was 150 in the original PR?

**Pull request merged**